source/Outfit.cpp

```diff
@@ -195,7 +195,104 @@ void Outfit::Load(const DataNode &node)
 
 	for(const DataNode &child : node)
 	{
-		if(child.Token(0) == "category" && child.Size() >= 2)
+		if(child.Token(0) == "clear" && child.Size() >= 2)
```

Comment:
`"clear"` is deprecated syntax, no need to add support for it.

Reply:
yeah but that said it should be replaced by "remove" without specifying what to remove
I kinda like the idea to be able to remove just one thing

Reply:
`remove` followed by one token does what `clear` was doing.